css/cv.css

```diff
@@ -0,0 +1,11 @@
+.main_page_button{
```

Comment:
Jeśli używasz tych samych klas to lepiej jest pisać 1 raz, czyli 1 cv.css i załączyć ten plik 2 razy.

Reply:
tutaj